cdap-distributions/src/COPYRIGHT/org.mockito:mockito-inline:4.11.0/COPYRIGHT

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
Test dependencies licenses need not be included in the licenses.

Reply:
As discussed, not a blocker.